README.md

```diff
@@ -206,4 +206,54 @@ This work would not have been possible without amazing open source models and da
 - DocLayNet from IBM
 - ByT5 from Google
 
-Thank you to the authors of these models and datasets for making them available to the community!
\ No newline at end of file
+# How to use this library for Python 3.10
+```shell
+pip install -r https://raw.githubusercontent.com/VikParuchuri/marker/master/requirements.txt
+```
+```shell
+    import os
+    
+    from marker.convert import convert_single_pdf
+    from marker.logger import configure_logging
+    from marker.models import load_all_models
+    from marker.output import save_markdown
+    
+    
+    def convert_pdf_to_markdown(filename, output_dir, max_pages=470, start_page=1, batch_multiplier=1, langs=["en"]):
+        """
+      Converts a PDF file to Markdown format using the Marker library.
+    
+      Args:
+          filename (str): Path to the PDF file.
+          output_dir (str): Directory to save the converted Markdown file.
+          max_pages (int, optional): Maximum number of pages to convert. Defaults to 10.
+          start_page (int, optional): Page number to start conversion from. Defaults to 1.
```

Comment:
Should the start page not default to 0?

I had an error when trying this with a single page pdf until I changed start_page from 1 to 0